src/components/InputDeviceConfig/AudioVideo.vue

```diff
@@ -117,13 +138,27 @@ export default {
         this.$modV.setupMedia({ videoId: value });
         this.switchingVideo = false;
       }
+    },
+
+    maxGain() {
+      return this.$modV.gainNode.gain.maxValue.toPrecision(1 + 2).split("e")[0];
+    },
+
+    minGain() {
+      return 0;
```

Comment:
We don't use the minValue from the gain node as it returns a negative value.
Negative gain inverts the signal, which is confusing and not expected for a gain slider.